data/navy watcher subplot

```diff
@@ -0,0 +1,673 @@
+			`	Rolling his eyes, Frank says "As soon as I told you about the Navy's super-secret plan to spy on the pirates. Jeez, yes. Anyway, we can give you twenty thousand for three tons. That's a pretty sweet deal, right? So, can we bring back some more of those awesome ferns?"`
+			choice
+				`	"At that price? Absolutely."`
+					accept
+				`	"The Navy's what now?"`
+			`"That research equipment? It's three tons of microscopic surveillance drones. We're supposed to dump 'em out the airlock on descent to the planet. They're like spores, except they have their own servos, so they can stay aloft much longer than mushroom spunk. Something about the pirates we ran into last time; apparently, they're as interested in Watcher's ferns as we are. Mostly for the same reasons, probably. So, we got a deal?"`
+			choice
+				`	"Sure."`
+					accept
+				`	"This is all getting a bit too shady for me."`
+					decline
+
+mission "Poachers 3"
+	landing
+	name "Poachers on Watcher"
+	description "Return Frank and his ferns to <planet>."
+	source Watcher
+	destination Hermes
+	passengers 1
+	cargo "psychoactive ferns" 3
+	to offer
+		has "Poachers 2: done"
+	to fail
+		has "chosen sides"
+	on offer
+		conversation
+			`While you descend, Frank disperses the research equipment into the lower atmosphere. You then spend the next several hours helping him cut and load your ship with more huge, delicate fern fronds.`
+				accept
+	npc
+		personality nemesis waiting
+		government Pirate
+		fleet "Small Southern Pirates" 2
+
+	on complete
+		payment
+		payment 60000
+		dialog
+			`You wish Frank good luck with his research on the psychoactive ferns. Between the Navy and Dr. Moore's university, you walk away with an extra <payment> in your account.`
+
+mission "Outpost on Watcher 1"
+	landing
+	name "Commander Moore's mystery delivery"
+	description "Visit Commander Moore on <planet> before <date>."
+	deadline 14
+	source
+		government Republic
+		attributes "near Earth" paradise north deep
+	destination
+		distance 2 6
+		near Algieba 2 6
+		government Republic
+		attributes factory
+	to offer
+		not "chosen sides"
+		has "event: war begins"
+		has "Poachers 3: done"
+		random < 35
+	to fail
+		has "chosen sides"
+	on offer
+		dialog
+			`You receive a message from Naval Commander Henry Moore. "The Navy has a delivery job if you're interested. I can't give you any details right now, but I can only hold on to this job for another two weeks. If you're interested, come visit me on <planet> before <day>."`
+	on fail
+		event "watcher watchout" 180
+	on decline
+		event "watcher watchout" 180
```

Comment:
Same as the recon missions (empty dirt belt, southern pirate home worlds, and find the pirate fleet) in the free worlds - without an explicit "accept/decline", but because it's in dialog, it automatically gives "accept/decline" buttons.